microsoft/testsuites/core/lsvmbus.py

```diff
@@ -121,7 +136,7 @@ def lsvmbus_count_devices_channels(self, node: Node) -> None:
         """,
         priority=4,
         requirement=simple_requirement(
-            supported_platform_type=[AZURE],
+            supported_platform_type=[AZURE], unsupported_os=[BSD]
```

Comment:
It should add Windows, because the default value doesn't include Windows.

Reply:
Is this applicable for future testcases we migrate too?

Reply:
yes